pkg/ctl/functions/create.go

```diff
@@ -420,21 +424,16 @@ func doCreateFunctions(vc *cmdutils.VerbCmd, funcData *util.FunctionData) error
 
 	admin := cmdutils.NewPulsarClientWithAPIVersion(common.V3)
 
-	if utils.IsPackageURLSupported(funcData.Jar) {
-		err = admin.Functions().CreateFuncWithURL(funcData.FuncConf, funcData.Jar)
-		if err != nil {
-			cmdutils.PrintError(vc.Command.OutOrStderr(), err)
-		} else {
-			vc.Command.Printf("Created %s successfully\n", funcData.FuncName)
-		}
+	if utils.IsPackageURLSupported(funcData.UserCodeFile) {
```

Comment:
what if a local executable's name starts with "http", like "http_test_function" as `-go` argument, will this local file wrongly triggered `CreateFuncWithURL` instead of `CreateFunc`?

Reply:
I believe that is correct, yes. That is, however, a pre-existing bug, so maybe it makes sense to solve that in a separate issue? It's probably just as simple as changing the prefix to require `http[s]://` and `file://`, but I am not entirely sure.

Reply:
Yes, i think include `"://"` to the prefix will solve this problem. A separate issue will be good idea.

Reply:
#264